### DISCUSSION TITLE
V9: Design choice Input Components resembling Google Material Design!

### DISCUSSION THREAD

**Discussion by blendsdk** [microsoft/fluentui#25796] [Q&A]
Hi,

I was wondering why the Input components in V9 have an activation indicator that is very much Material Design?
If these are by intention then please consider removing the lower `border-radius` of these components.

- [V9 Input](https://react.fluentui.dev/?path=/docs/components-input--default)
- [Just look, don't use: :smile:](https://m2.material.io/components/text-fields#anatomy)

**Comment by JustSlone** [comment]
It's not intentional to mimic Material Design here, the v9 default styling is part of the new Fluent Design look and feel (public UI kits and designs coming soon!). 

I'll ask though, what is the reason for wanting to remove the lower `border-radius`?

**Reply by blendsdk** [reply]
The bottom-activation-indicator-line, when a field is in focus, breaks the bottom border-radius. Please note: this is going to be difficult to fix. Unless you use fancy SVG layering, (like done in MaterialDesign)

**Reply by smhigley** [reply]
I'm seeing the border radius preserved when focused -- @blendsdk are you getting something different than this?
![screenshot of focused field, with blue underline focus indicator following the bottom border radii but with a flat top](https://user-images.githubusercontent.com/3819570/209186976-0baa6fd5-f00c-4ae0-bbd4-f04fbcde1d2f.png)

The flat top of the focus indicator was an intentional design choice (that initially confused me when first implementing it 😅). If you're seeing something different than the screenshot, could you let us know the browser version and OS?
